CONTRIBUTING.md

```diff
@@ -35,23 +35,17 @@ Any module must follow this basic format:
 ```js
 module.exports = function ModuleName(options,UI) {
 
-  options = options || {};
```

Comment:
Here let's say `Module requirements have simplified in version XXX` - see here for previous (and link to this section of the readme from a specific release -- like https://github.com/publiclab/image-sequencer/blob/master/CONTRIBUTING.md#contributing-modules

Reply:
@jywarren Got it!! Also since the UI object is still being passed to the module, it can technically do anything with it!!